stack.yaml

```diff
@@ -64,7 +64,7 @@ extra-deps:
   # a version > 1.0.0 of wai-middleware-prometheus is available
   # (required: https://github.com/fimad/prometheus-haskell/pull/45)
 - git: https://github.com/wireapp/saml2-web-sso
-  commit: 687d9ac8ac2994aff8436189c6ecce29faad8500  # master (May 18, 2020)
+  commit: be04baaffb92ff64d8e1d9f400ec45849d754faf  # akshaymankar/ghc-8.8 (Aug 21, 2020)
```

Comment:
The various forks should have a reference to a stable branch, not a PR branch which will be deleted after merging the PR. Perhaps before merging this PR; merge the upstream ones and use a stable branch commit identifier?

Reply:
This one is the only one we control. I created PR here: https://github.com/wireapp/saml2-web-sso/pull/66
No other branches will get deleted on merge, I have set the option to not delete it in Gitlab. For github forks we don't need to worry as github only deletes branches in the same repo. 

I suspect some of the upstream maintainers aren't maintaining the packages very actively, so it might take very long. So, I think it is fine to refer to these packages in their forks.